### PR TITLE
Update LocalVMSetup.md

### PR DESCRIPTION
**Pull request by hessio**
I added some steps on how to configure your network settings.

### CODE REVIEW

File: doc/LocalVMSetup.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ THIS FILE IS OLD AND SHOULD BE REMOVED AFTER VERIFYING THAT NO LINKS EXIST TO IT
 
 # Create a new VM from scratch
 For CORTX development, you can create a new VM from scatch and then install a fresh CentOS 7.8.2003.
+This setup should take you roughly 20 to 30 minutes
```
Comment (andriytk) [inline doc/LocalVMSetup.md:6]: Missed dot at the end of the sentence.

File: doc/LocalVMSetup.md [hunk 2]
```diff
@@ -16,9 +17,29 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
...
+   
+if you have a network connection then skip on to step 4, otherwise follow the below commands:
+
+     sudo nmtui
+ 
+ -> Edit a connection, select your network interface and choose "Automatically connect" option (by pressing the space key) and then select OK.
+ 
+    sudo reboot now
+
+## 5. Connect to your VM using Putty
```
Comment (andriytk) [inline doc/LocalVMSetup.md:33]: Putty is spelled "PuTTY" correctly (according to [their site](https://www.chiark.greenend.org.uk/~sgtatham/putty/)). But I don't think we should limit users to PuTTY only. We can only suggest it.

File: doc/LocalVMSetup.md [hunk 3]
```diff
@@ -16,9 +17,29 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
...
+Get the ip address of your VM sudo nmcli will provide you with the ip address. Copy the ip address to your Putty console and click connect.
+
+## 6. Install git
+
+    yum insatll git
```
Comment (andriytk) [inline doc/LocalVMSetup.md:38]: Looks like typo in the command.

File: doc/LocalVMSetup.md [hunk 4]
```diff
@@ -16,9 +17,29 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
...
+if you have a network connection then skip on to step 4, otherwise follow the below commands:
+
+     sudo nmtui
+ 
+ -> Edit a connection, select your network interface and choose "Automatically connect" option (by pressing the space key) and then select OK.
+ 
+    sudo reboot now
+
+## 5. Connect to your VM using Putty (Optional)
+Get the ip address of your VM sudo nmcli will provide you with the ip address. Copy the ip address to your PuTTY console and click connect.
```
Comment (huanghua78) [inline doc/LocalVMSetup.md:34]: * This step says "connect to your VM", but does not say what to do after connecting to your VM.

* The "Optional" is confusing. The PuTTY is optional? This step is optional?

* We are not assuming people using MS Windows platform. 
   If this step is necessary, please modify it as:
   Connect to your VM using your favorite ssh client.

```
Get the ip address of your VM.  *sudo nmcli*  will provide you with the ip address.
```

File: doc/LocalVMSetup.md [hunk 5]
```diff
@@ -16,9 +17,29 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
...
+
+## 6. Install git
+
+    yum install git
```
Comment (huanghua78) [inline doc/LocalVMSetup.md:38]: * This is not a MUST-HAVE. If people are trying to install CORTX from PRMs, they don't need "git".
* If sudo is needed in modified step 4, then sudo is needed here, too.

**Comment by shailesh-vaidya** [comment]
@hessio Thanks for your contribution. Please fix [DCO check](https://github.com/Seagate/cortx/pull/560/checks?check_run_id=1700514046) reported by DCO bot.

File: doc/LocalVMSetup.md [hunk 6]
```diff
@@ -16,9 +17,29 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
 ## 3. Install a fresh CentOS 7.8.2003
 Automatic install or manual install. Basic installation is OK. When you build CORTX from source in next steps, dependant packages will be resolved and installed.
 
-## 4. You may need to add EPEL repo.
+## 4. You may need to configure your network settings
+run a test to check if you are connected to the network 
+
+     sudo nmcli d
```
Comment (andriytk) [inline doc/LocalVMSetup.md:23]: Use markup for the commands:

```sh
sudo nmcli d
```

**Review by andriytk — CHANGES_REQUESTED** [review]

**Comment by andriytk** [comment]
Please fix the issues mentioned by @huanghua78 also.

File: doc/LocalVMSetup.md [hunk 7]
```diff
@@ -16,9 +17,24 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
...
 Automatic install or manual install. Basic installation is OK. When you build CORTX from source in next steps, dependant packages will be resolved and installed.
 
-## 4. You may need to add EPEL repo.
+## 4. You may need to configure your network settings
+run a test to check if you are connected to the network 
+
+```sh
+sudo nmcli d
+```   
+if you have a network connection then skip on to step 4, otherwise follow the below commands:
```
Comment (andriytk) [inline doc/LocalVMSetup.md:26]: Which step 4?

File: doc/LocalVMSetup.md [hunk 8]
```diff
@@ -16,9 +17,24 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
 ## 3. Install a fresh CentOS 7.8.2003
 Automatic install or manual install. Basic installation is OK. When you build CORTX from source in next steps, dependant packages will be resolved and installed.
 
-## 4. You may need to add EPEL repo.
+## 4. You may need to configure your network settings
+run a test to check if you are connected to the network
```
Comment (andriytk) [inline doc/LocalVMSetup.md:21]: Start the sentence with a capital letter. Consider adding a colon `:` at the end here.

**Review by andriytk — CHANGES_REQUESTED** [review]

**Review by andriytk — APPROVED** [review]
Seems OK.

**Comment by mukul-seagate11** [comment]
@all-contributors please add @hessio for network

**Comment by mukul-seagate11** [comment]
@all-contributors please add @hessio for network

**Comment by mukul-seagate11** [comment]
@all-contributors please add @hessio for code

**Comment by mukul-seagate11** [comment]
Seems, https://github.com/Seagate/cortx/pull/546 is duplicate PR

**Comment by hessio** [comment]
I feel as though we should close this as I made this within my few weeks starting here and I don't believe it is useful, nor does it make that much sense.

**Pull request closed without merging**